lib/src/default_extension_map.dart

```diff
@@ -295,12 +295,13 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'grv': 'application/vnd.groove-injector',
   'grxml': 'application/srgs+xml',
   'gsf': 'application/x-font-ghostscript',
-  'gtar': 'application/x-gtar',
+  'gtar': 'application/gzip',
   'gtm': 'application/vnd.groove-tool-message',
   'gtw': 'model/vnd.gtw',
   'gv': 'text/vnd.graphviz',
   'gxf': 'application/gxf',
   'gxt': 'application/vnd.geonext',
+  'gz': 'application/gzip',
```

Comment:
Added the standard GZip files to the mapping

Reply:
I think this one is easy to add since it's official.
https://www.iana.org/assignments/media-types/application/gzip